tests/js/spec/stores/organizationStore.spec.jsx

```diff
@@ -68,4 +70,41 @@ describe('OrganizationStore', function() {
       dirty: false,
     });
   });
+
+  it('loads in sorted teams', async function() {
+    const organization = TestStubs.Organization();
+    OrganizationActions.update(organization);
+    // wait for action to get dispatched to store
+    await tick();
```

Comment:
Would it make sense to check that `organization.teams` is undefined or an empty array after the tick?  

My reasoning is that if someone were to load Org+Team in the future, this would break and alert them that they shouldn't do it.

Reply:
spoke with danny, should be resolved